ui/lib/core/addon/components/navigate-input.hbs

```diff
@@ -1,6 +1,7 @@
 <div class="navigate-filter">
   <div class="field" data-test-nav-input>
     <p class="control has-icons-left">
+      {{! template-lint-disable no-down-event-binding }}
```

Comment:
my linter was failing on this, which I think was removed in the recent merge of main into this branch.